src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -7252,428 +7270,422 @@ private void makeCoilRecipes() {
-                            GT_Utility.getIntegratedCircuit(3) },
-                    Materials.Lava.getFluid(10000L),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_Ne", 1L, 0),
-                    300,
-                    30);
-            // End
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "end_stone", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "obsidian", 64L, 0),
-                            GT_ModHandler.getModItem(Natura.ID, "Cloud", 64L, 1), GT_Utility.getIntegratedCircuit(4) },
-                    FluidRegistry.getFluidStack("ender", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_ED", 1L, 0),
-                    300,
-                    30);
-            // Far End Asteroids
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 1),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 2),
-                            GT_Utility.getIntegratedCircuit(5) },
-                    FluidRegistry.getFluidStack("endergoo", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_EA", 1L, 0),
-                    300,
-                    30);
+        // T0 Planets
+        // Overworld
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        getModItem(Botania.ID, "tinyPlanetBlock", 1, 0),
+                        getModItem(Minecraft.ID, "stone", 64, 0),
+                        getModItem(Minecraft.ID, "grass", 64, 0),
+                        getModItem(Minecraft.ID, "sapling", 1, 0),
+                        GT_Utility.getIntegratedCircuit(1))
+                .fluidInputs(Materials.Water.getFluid(10000))
+                .itemOutputs(getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_Ow", 1, 0)).noFluidOutputs()
+                .duration(15 * SECONDS).eut(TierEU.RECIPE_LV).addTo(sAssemblerRecipes);
+        // Twilight Forest
+        if (TwilightForest.isModLoaded()) {
+            GT_Values.RA.stdBuilder()
+                    .itemInputs(
+                            getModItem(Botania.ID, "tinyPlanetBlock", 1, 0),
+                            getModItem(Minecraft.ID, "stone", 64, 0),
+                            getModItem(Minecraft.ID, "grass", 64, 0),
+                            getModItem(TwilightForest.ID, "tile.TFSapling", 1, 1),
+                            GT_Utility.getIntegratedCircuit(2))
```

Comment:
small problem: the Twilight Forest recipe is broken (probably already before your change). It conflicts with stone->pressure plate which is also on circuit 2

Reply:
It seems to me like the recipe for the pressure plate can be changed to circuit 3 without issue. Is that an acceptable solution?

Reply:
Some thought and investigation definitely needs put into making these not conflict, I don't see a super clear answer without digging deeper and probably having to change the other recipes that conflict with it. The recipe for The End one is also conflicting because of the obsidian. Should I try to tackle that as part of this PR or should I make a new issue for these? These were present even before the conversion of the recipes to RA2.

The easy answer is to make them not use stone for TF and not use obsidian for The End, but I'm not sure what suitable replacement blocks might be for those.

Reply:
I would leave the pressure plate as its pretty basic and comes from like 20 stone types. the TF thing is super specific and can be on any some other circuit no problem

Reply:
the circuits overall look very strange. no reason to go through all numbers. Spoke with GDC that made the recipe. Proposal would be to put _all_ the planet recipes on circuit 17. should be safe and makes more sense.

Reply:
I converted them all to circuit 17, none show up in the conflicting recipes now.